docs/packaging/packaging-changes.md

```diff
@@ -0,0 +1,51 @@
+# Packaging Changes
+
+This document is meant as a changelog of sorts for packaging practices. Refer to specific help pages for more details.
+
+- This list is not exhaustive
+- This list is in _reverse chronological order_ (newest first)
+
+## Upcoming
+
+_These items are not mandatory yet, and are subject to change_
+
+#### `makefile` -> `go-task`
```

Comment:
This should probably be included in the September changes, as it's likely to be landed this month. See https://github.com/getsolus/packages/pull/346 and #375.

Reply:
moved